code/game/machinery/bots/secbot.dm

```diff
@@ -242,7 +246,7 @@ Auto Patrol: []"},
 
 				else								// not next to perp
 					var/turf/olddist = get_dist(src, target)
-					walk_to(src, target,1,4)
+					walk_to(src, target, 1, 4 * (fast == 1 ? fast_delaymod : 1))
```

Comment:
The delaymod variable is a modifer, so 0.4 would mean the delay is only 40% usual. I can replace it with a "boostdelay" and give it a value of 1.6 which is the same result